gitspy/User.swift

```diff
@@ -8,9 +8,36 @@
 
 import Foundation
 
-struct User {
+struct User: Parseable {
     
     let id: Int
     let name: String
     
+    // MARK: Network
+    
+    static func me(token: String, completion: (user: User?) -> Void) {
+        let userResource = Resource(pathComponent: "me") { (object) -> User? in
+            // TODO: move this in init?(dictionary)  ??
```

Comment:
I think there might be two APIs, one with the closure to parse the response, for non-parseable objects (like Bool)
The other one don't need the closure since we can parse Parseable objects by initializing it with the reply